driverconfig/config.yaml

```diff
@@ -63,6 +63,7 @@ drivers:
           - version: v121
           - version: v122
           - version: v123
+          - version: v124
```

Comment:
@panigs7, please remove the entry for v2.0.0 under unity in the next PR as this PR is merged

Reply:
sure